config/routes.rb

```diff
@@ -1,15 +1,8 @@
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
   resources :merchants do
-    resources :items, controller: 'merchant_items'
-    resources :dashboard, controller: 'merchant_dashboard'
+    resources :items, controller: 'merchant_items', only: [:index, :show]
+    resources :dashboard, controller: 'merchant_dashboard', only: [:index]
+    resources :invoices, controller: 'merchant_invoices', only: [:index]
```

Comment:
I'm still curious about these controllers; I'm gonna make a note to ask about it at the check in.